home10.1/FootballTournament.java

```diff
@@ -0,0 +1,73 @@
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Random;
+import java.util.Scanner;
+
+public class FootballTournament {
+
+    public static void inputTeams(List<Team> teams, Scanner scanner) {
+        for (int i = 0; i < 4; ++i) {
```

Comment:
4 can be a field of this class either constant or passed as a constructor argument